presto-druid/src/test/java/com/facebook/presto/druid/TestDruidConfig.java

```diff
@@ -28,10 +29,11 @@
     public void testDefaults()
     {
         assertRecordedDefaults(recordDefaults(DruidConfig.class)
-                    .setDruidBrokerUrl(null)
-                    .setDruidCoordinatorUrl(null)
-                    .setDruidSchema("druid")
-                    .setComputePushdownEnabled(false));
+                .setDruidBrokerUrl(null)
+                .setDruidCoordinatorUrl(null)
+                .setDruidSchema("druid")
+                .setComputePushdownEnabled(false)
+                .setHadoopResourceConfigFiles((String) null));
```

Comment:
why ((String) null)?

Reply:
Because there is another setter as below:
`    public DruidConfig setHadoopConfiguration(List<String> files)
    {
        if (files != null) {
            this.hadoopConfiguration = ImmutableList.copyOf(files);
        }
        return this;
    }`

We need to provide to type explicitly.    Remove the redundant setter above, can we?   I just follow the pattern in presto-hive's config